server/Controllers/contestControllers.js

```diff
@@ -124,9 +124,32 @@ async function getContests(req, res) {
   }
 }
 
+async function addResultToContest(req, res, resultId) {
+  try {
+    const contest = await Contest.findById(req.body.contestId);
+    const obedienceClass = contest.obedienceClasses.find(
+      (obedienceClass) => obedienceClass.classNumber == req.body.obedienceClass,
+    );
+    const participant = obedienceClass.participants.find(
+      (participant) =>
+        participant.participantId.valueOf() === req.body.participantId,
+    );
+    participant.resultsId = resultId;
+    const updatedContest = await contest.save();
+    if (!updatedContest) {
+      res.send(500).end();
```

Comment:
Some reading about res.end(), it might not be necessary:
https://stackoverflow.com/questions/29555290/what-is-the-difference-between-res-end-and-res-send